src/Controls/src/Core/Window/Window.cs

```diff
@@ -410,6 +418,42 @@ void SendWindowDisppearing()
 			IsActivated = false;
 		}
 
+		public void SetTitleBar(TitleBar? titleBar)
```

Comment:
Why do we need this method if we can add things to the property changed delegate in the bindable property? See `PageProperty`